storefront/src/app/core/config/public-tenant-config.service.ts

```
@@ -0,0 +1,88 @@
+            );
+    }
+
+    clearCache(subdomain?: string): void {
+        if (subdomain) {
+            this.#configCache.delete(subdomain);
+            sessionStorage.removeItem(this.#STORAGE_PREFIX + subdomain);
+        } else {
+            this.#configCache.clear();
+            Object.keys(sessionStorage)
+                .filter(key => key.startsWith(this.#STORAGE_PREFIX))
+                .forEach(key => sessionStorage.removeItem(key));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)